instrumental/drivers/__init__.py

```diff
@@ -452,34 +452,40 @@ def write(self, message, *args, **kwds):
 
         >>> inst.write('source{}:value {}', channel, value)
         """
-        full_message = message.format(*args, **kwds)
```

Comment:
Perhaps this could be refactored in some way to avoid duplication with the code below in query()